lib/engines/aws.rb

```diff
@@ -92,5 +94,87 @@ def translate(input_text, src_lang, target_lang)
       :source_language_code => "#{src_lang}", :target_language_code => "#{target_lang}"})
     response.translated_text
   end
+
+  def transcribe_uri(video_uri, lang_code, bucket_name)
```

Comment:
Should we not define these APIs in translator.rb and implement here in this Engine ?